src/main/scala/org/broadinstitute/dsde/workbench/sam/api/ResourceRoutes.scala

```diff
@@ -37,53 +44,53 @@ trait ResourceRoutes extends UserInfoDirectives {
     pathPrefix("resource") {
       requireUserInfo { userInfo =>
         pathPrefix(Segment / Segment) { (resourceTypeName, resourceId) =>
-          val resourceType = resourceTypes.getOrElse(ResourceTypeName(resourceTypeName), throw new WorkbenchExceptionWithErrorReport(ErrorReport(StatusCodes.NotFound, s"resource type $resourceTypeName not found")))
```

Comment:
the rest of this file is white space changes except this line is replaced with a call to withResourceType